src/main/java/EventTask.java

```diff
@@ -0,0 +1,18 @@
+public class EventTask extends Task {
+
+    private String eventTime;
+
+    public EventTask(String description) {
+        super(description.split(" /")[0]);
+        String[] output = description.split("/");
+        String pattern = ("(at?)(\\s)(.+)");
+        eventTime = output[1].replaceAll(pattern, "$3");
+    }
+
+    @Override
+    public String toString()
+    {
+        return String.format("[E][%s] %s (at: %s)", getStatusIcon(), description, eventTime);
```

Comment:
Maybe the type of the event represented by the "E" here should be static since it's a constant?